script/install-as-user

```diff
@@ -127,6 +127,9 @@ sed -r \
     -e "s,# constraint_disabling: false,  constraint_disabling: false," \
     config/database.yml-example > config/database.yml
 
+# create the template_utf8 template we'll use for our databases
+createdb -T template0 -E UTF-8 template_utf8
```

Comment:
I wonder if we should also be setting `datallowconn=false` to prevent connections to it.